pinot-hadoop/src/main/java/org/apache/pinot/hadoop/PinotHadoopJobLauncher.java

```diff
@@ -29,7 +30,8 @@
 public class PinotHadoopJobLauncher {
```

Comment:
may be nice to also add an option to do preprocessing and other job types too

Reply:
i see, talked offline, in open source we don't technically support this yet because we have to change some parameters to make that work - can we create a small ticket somewher eto add that functionality?